+jrclust/+views/plotAuxCorr.m

```diff
@@ -107,12 +107,33 @@
             auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
         case {'.dat', '.bin'}
             if isempty(hCfg.auxChan)
-                return;
+                try % ask where it is 
+                    hCfg.auxFile = fullfile(hCfg.outputDir,uigetfile({'*.mat;*.csv'},...
+                    'Select the Aux file',hCfg.outputDir));
+                catch
+                    return;
+                end
             end
-
-            hRec = jrclust.detect.newRecording(hCfg.auxFile, hCfg);
-            auxSamples = single(hRec.readRawROI(hCfg.auxChan, 1:hRec.nSamples))*hCfg.bitScaling*hCfg.auxScale;
-            auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
+            try
+                hRec = jrclust.detect.newRecording(hCfg.auxFile, hCfg);
+                auxSamples = single(hRec.readRawROI(hCfg.auxChan, 1:hRec.nSamples))*hCfg.bitScaling*hCfg.auxScale;
+                auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
+            catch % data not from SpikeGLX
+                auxData = load(hCfg.auxFile);
+                auxDataFields = fieldnames(auxData);
+                auxRateFieldIdx= ~cellfun(@isempty, cellfun(@(fn) strfind(fn,'Rate') |...
+                    strfind(fn,'sampling'), auxDataFields,'UniformOutput', false));
+                if sum(auxRateFieldIdx) %there's a sampling rate field 
```

Comment:
You should use `any` here instead of `sum`. `any` will stop and return `true` at the first `true` it encounters.

Reply:
Old habits die hard I guess. I realized that and started updating code lines with this but many slipped through I suppose.